### ISSUE TITLE
Extension accidentaly crashed while i was refactoring my endpoints

### ISSUE THREAD

**Issue by Nour-eddineAE** [a85/POSTMan-Chrome-Extension#12220]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

![image](https://github.com/postmanlabs/postman-app-support/assets/88547393/5bd01431-ed93-40b6-b025-15b11c79de77)

"As you can see above, I have some endpoints, and I wanted to group them in folders. However, when I finished creating folders, the extension crashed, and even the logs provided no helpful data."

### Steps To Reproduce

All i was doing was to create folders, inside a collection, when i finished i wanted to add endpoints inside the folders, then the extension crashed

### Screenshots or Videos

![image](https://github.com/postmanlabs/postman-app-support/assets/88547393/5bd01431-ed93-40b6-b025-15b11c79de77)

### Operating System

Windows

### Postman Version

vscode postman extension, version 0.5.4

### Postman Platform

Other

### User Account Type

Signed In User

```[tasklist]
### Tasks
```

**Comment by kartikjain26** [comment]
@Nour-eddineAE Thanks for reaching out. We'll triage this and keep you updated.
Are you facing this issue repeatedly ?

To confirm your steps:
1. Create a collection
2. Added multiple folders
3. Try to add request/APIs in a folder led to this crash

Is that correct ?

**Comment by Nour-eddineAE** [comment]
@kartikjain26 I've installed the extension 5 days ago, and today was the first and only time this problem happened. I lost the bug when I reloaded vscode (as I had no more options, the screenshot shows what I was able to see).
The steps that led to the crash:
1. I already have the collection as I was using postman app before so I didn't create it this time.
2. I created a couple of folders inside the collection
3. I was trying to drag and drop the endpoints inside the folders(endpoints were not grouped in folders, I created the folders to group them)
4. After the drag and drop didn't work I started recreating the endpoints, when I clicked the first folder this problem happened.
